Modules/Core/Common/include/itkShapedFloodFilledFunctionConditionalConstIterator.hxx

```diff
@@ -80,7 +80,7 @@ ShapedFloodFilledFunctionConditionalConstIterator<TImage, TFunction>::Initialize
   m_ImageRegion = this->m_Image->GetBufferedRegion();
 
   // Build and setup the neighborhood iterator
-  auto radius = MakeFilled<typename NeighborhoodIteratorType::RadiusType>(1);
+  constexpr auto radius = MakeFilled<typename NeighborhoodIteratorType::RadiusType>(1);
```

Comment:
We could even declare those variables `static constexpr`, instead of just non-static `constexpr`. I don't know if it matters much, in these cases 🤷 Some people prefer to always declare local variables `static constexpr` if possible.